src/ROUTES.js

```diff
@@ -17,7 +17,10 @@ export default {
     IOU_BILL: 'iou/split',
     SEARCH: 'search',
     SIGNIN: 'signin',
-    SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:validateCode',
+    SET_PASSWORD_WITH_VALIDATE_CODE: 'setpassword/:accountID/:validateCode',
```

Comment:
I'm not sure we need to do anything about this. But it would be good to start thinking about how we would handle such a change in the future. 

e.g. we are breaking any existing links sitting in people's inboxes since the new app will treat `validateCode` as `accountID` and if they get a new deep link but still have the old app then we will have the opposite problem.

Reply:
I think it would be a good idea to create some kind of issue to discuss a solution for problems like this just to get it on people's radar.